header.php

```diff
@@ -132,4 +132,4 @@
 		<?php 
 		// Output the menu modal
 		get_template_part( 'parts/modal-menu' ); 
-		?>
\ No newline at end of file
+		?>
```

Comment:
There's no reason to have this closing tag, and looking at Twenty Nineteen, it looks like the convention is to omit it at the end of a file.

Reply:
@ZebulanStanphill I didn't add that, just changed the protocol :)

Reply:
Oh, I see. The closing tag was already there, but your editor added a newline character.

Reply:
It's directly via GH, no idea :)

Reply:
This repo has a `.editorconfig` that specifies having a line break at the end of every file, and apparently GitHub has native support for EditorConfig:

https://github.com/editorconfig/editorconfig.github.com/pull/48

So that's probably where it came from.